README.md

```diff
@@ -44,6 +59,7 @@ nixpkgs.overlays = [
     nix-direnv = super.nix-direnv.override {
       nix = super.nix-monitored;
     };
+    # Line above here if you want it.
```

Comment:
I believe this entire section is obsolete if you use the module, and can be removed.